os/drivers/audio/alc5658.c

```diff
@@ -597,6 +597,21 @@ static int alc5658_configure(FAR struct audio_lowerhalf_s *dev, FAR const struct
 				ret = -EDOM;
 		}
 		break;
+
+		case AUDIO_FU_MUTE: {
+			/* Mute or unmute:  true(1) or false(0) */
+
+			bool mute = caps->ac_controls.hw[0];
```

Comment:
to use bool type, you should include <stdbool.h>

Reply:
we can use b[0] as a uint8_t instead of uint16_t that's why it defined as union, I think.

Reply:
@Taejun-Kwon can you elaborate? Yes, either b[0] or hw can be used, Do you mean uint8_t is compatible to char(bool)?
If so, then yes, it makes sense to modify. If anything else, please expalin.